OrbitGl/App.cpp

```diff
@@ -260,8 +259,13 @@ void OrbitApp::PostInit() {
   }
 
   if (!options_.grpc_server_address.empty()) {
-    grpc_channel_ = grpc::CreateChannel(options_.grpc_server_address,
-                                        grpc::InsecureChannelCredentials());
+    grpc::ChannelArguments channel_arguments;
+    // The default receive message size is 4mb. Symbol data can easily be more
+    // than this. -1 means no size limit
+    channel_arguments.SetMaxReceiveMessageSize(-1);
```

Comment:
Should we still have some kind of limit, even if very high?

Reply:
I set a limit of 400mb and wrote a comment explaining it